docs/source/contributing.md

```diff
@@ -1,14 +1,30 @@
-Contributing to the Leapp project
-=================================
+# Contributing to the Leapp project
 
 First, thank you for taking your time to contribute to the project.
 
 The following is a set of guidelines for contributing effectively to the Leapp-related repositories
 hosted under the `OS and Application Modernization Group organization <https://github.com/oamg/>`_
 on GitHub.
 
-Submitting a Pull Request
-^^^^^^^^^^^^^^^^^^^^^^^^^
+## Code style guidelines
+
+Your code should follow our
+[Python Coding Guidelines](python-coding-guidelines.html)
+
+## Best practices in actor development
+
+- actor development: [Best practices for actor development](best-practices.html)
+- actor directory layout might be found in 
+[Repository directory layout](repository-dir-layout.html)
+(under `actorname/`)
+- actor testing: [Testing actors](test-actors.html)
+
+## RPM packages naming schema
```

Comment:
I think that the way how are the Copr RPMs named does not relate to contributions to the project from others much. I'd maybe link to that `build-schema` page only from under https://leapp.readthedocs.io/en/latest/infrastructure.html.